packages/ui/src/proposals/helpers/getDetailsRenderStructure.ts

```diff
@@ -201,6 +202,16 @@ const booleanMapper: Mapper<UpdateChannelPayoutsDetail, 'channelCashoutsEnabled'
   ]
 }
 
+const palletStatusMapper: Mapper<UpdatePalletFrozenStatusDetail, 'frozen'> = (value) => {
+  return [
+    {
+      label: 'Proposed Status',
+      renderType: 'Text',
+      value: value ? 'Enable' : 'Disable',
+    },
+  ]
+}
```

Comment:
And I'm wondering if the current pallet state should be displayed too. WDYT @dmtrjsg @chrlschwb ?